### PR TITLE
obfuscation is now turnoffable

### PR DESCRIPTION
**Pull request by enola-dkfz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,13 +29,15 @@ API_KEY = "App1Secret"
...
-optional variable
+### Optional variables
+
 ```bash
 RETRY_COUNT = "32"
 ```
 
 The default maximum number of retries for beam and blaze healthchecks is 32. 
 
+To disable the count obfuscation, set the env. variable `DO_NOT_OBFUSCATE`.
```
Comment (lablans) [inline README.md:40]: Please give values for on / off.

Consider using a positive variable (negations are confusing)

File: src/config.rs [hunk 2]
```diff
@@ -42,6 +42,10 @@ struct CliArgs {
     #[clap(long, env, value_parser)]
     blaze_url: Uri,
 
+    /// Should the results not be obfuscated
+    #[clap(long, env, value_parser)]
```
Comment (lablans) [inline src/config.rs:46]: Don't you want to set default value == false (meaning "yes, obfuscate?)

negations are confusing

File: src/main.rs [hunk 3]
```diff
@@ -167,9 +167,14 @@ async fn run_cql_query(task: &BeamTask, query: &Query, obf_cache: &mut ObfCache)
         }
     };
 
-    let cql_result_new = obfuscate_counts_mr(&cql_result, obf_cache);
+    let cql_result_new = match CONFIG.do_not_obfuscate {
+        false => obfuscate_counts_mr(&cql_result, obf_cache)?,
+        true => cql_result
+    };
+
+    dbg!(cql_result_new.clone());
```
Comment (lablans) [inline src/main.rs:175]: Forgot to remove?

**Review by lablans — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 3 above)
Comment (enola-dkfz) [reply src/main.rs]: Removed

File: src/config.rs (hunk 2 above)
Comment (enola-dkfz) [reply src/config.rs:46]: When the parameter is not present, it is false, no default needed

File: README.md (hunk 1 above)
Comment (enola-dkfz) [reply README.md:40]: It doesn't work with boolean. 
https://github.com/clap-rs/clap/issues/1649
Do you want it to be a string?

Comment (lablans) [reply README.md:40]: Surprising but then I'm fine with it

**Review by TKussel — APPROVED** [review]
With all concerns of @lablans addressed, this looks fine.

**Pull request merged**